src/patternfly/components/Button/examples/Button.md

```diff
@@ -50,6 +50,10 @@ import './Button.css'
 {{#> button button--modifier="pf-m-inline pf-m-link"}}
   Inline link
 {{/button}}
+
+{{#> button button--modifier="pf-m-inline pf-m-plain"}}
+  Inline link
```

Comment:
Should this be an icon?

Reply:
Yes! Updated that and a couple of other things.